src/style.css

```diff
@@ -22,18 +24,100 @@
 
 @media (min-width: 768px) {
   body {
-    background-color: teal;
+    background-color: #f7f7f9;
     margin: 0;
     padding: 0;
     width: 100%;
     height: auto;
+    box-sizing: border-box;
   }
 
   .h1 {
     font-size: 2.5rem;
     font-weight: 700;
     font-family: serif;
     text-align: center;
```

Comment:
It's nice you have done using media queries, it would be nice if avoided repetition.

Reply:
Ok, thank you for the feedback. I will address it but I think the media query can use the same classes at the same time different classes if one chooses. However, I will look at it. Thank you again.